lib/Backend/BackwardPass.cpp

```diff
@@ -1818,6 +1908,15 @@ BackwardPass::ProcessBailOutInfo(IR::Instr * instr)
             BVSparse<JitArenaAllocator> * byteCodeUpwardExposedUsed = byteCodeUsesInstr->byteCodeUpwardExposedUsed;
             if (byteCodeUpwardExposedUsed != nullptr)
             {
+                //Stack Args for Formals optimization.
```

Comment:
Might want to mention here that if stack args opt is enabled, then the only thing that may require scope object downstream is formals access (which is why it is ok to clear the bit in byteCodeUpwardExposedUsed)